doc/lua_api.txt

```diff
@@ -7314,6 +7314,7 @@ child will follow movement and rotation of that bone.
     * `sky_parameters` is a table with the following optional fields:
         * `base_color`: ColorSpec, changes fog in "skybox" and "plain".
           (default: `#ffffff`)
+        * `body_orbit_tilt`: Float. Valid range [-60.0,60.0] (default: 0.0)
```

Comment:
```suggestion
        * `body_orbit_tilt`: Float, angle of sun/moon orbit in degrees, relative to Y axis.
           Valid range [-60.0,60.0] (default: 0.0)
```

Reply:
Adopted, pushed amended commit including the suggestion.